indexers/keyvalue/RedisDBIndexer/__init__.py

```diff
@@ -69,9 +69,16 @@ def query(self, key: int, *args, **kwargs) -> Optional[bytes]:
         with self.get_add_handler() as redis_handler:
             for _key in redis_handler.scan_iter(match=key):
                 res = {
-                  "key": _key,
-                  "values": redis_handler.get(_key),
+                    "key": _key,
+                    "values": redis_handler.get(_key),
                 }
                 result.append(res)
 
-        return result
\ No newline at end of file
+        return result
+
+    def update(self, keys: Iterator[int], values: Iterator[bytes]):
+        raise NotImplemented()
+
+    def delete(self, keys: Iterator[int]):
+        raise NotImplemented()
```

Comment:
I'm confused. Wasn't there already some work here?

Reply:
now I committed the correct files